docs/UserGuide.md

```diff
@@ -101,7 +101,9 @@ Examples:
 * `addPatient n/Ben Smith p/98353535 e/ben@abc.com ic/S1234567A a/1 Ben Street, block 13, #01-01` adds the patient `Ben Smith` to the list.
 * `addDoctor n/Sarah Tan p/99123456 e/sarah@abc.com ic/T7654321P a/Sarah Rd t/Pediatrician` adds the doctor `Sarah Tan` to the list.
 
-![AddPatient](images/AddPatient.png)
+![AddPatient](images/AddPatient_updated.png)
+
+![AddDoctor](images/AddDoctor.png)
```

Comment:
I believe we should add a few tags such as "Pediatrician, Cardiologist, etc." for doctors to showcase more features in our UG.